zxcvbn/index.d.ts

```diff
@@ -0,0 +1,205 @@
+// Type definitions for zxcvbn 4.4.0
+// Project: https://github.com/dropbox/zxcvbn
+// Definitions by: Daniel McAssey <https://github.com/DanielMcAssey>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function zxcvbn(password: string, userInputs?: Array<string>): zxcvbnResult;
```

Comment:
From the documentation this file is a module, see https://github.com/dropbox/zxcvbn#node--npm--meteorjs. i.e. `import zxcvbn = require("zxcvbn");` should work.

to make this happen you need to add

```ts
export = zxcvbn;   // make it a module
export as namespace zxcvbn; // make sure there is a global function called zxcvbn as well
```